docs/tutorial.md

```diff
@@ -133,20 +145,38 @@ Now when you save and run this code, you should finally see something:
 So far we have only worked with numbers, but Javascript can work with more variable types:
 
 - A string is a piece of text that can contain multiple characters. Strings are defined by using the quotes `''`
+
+```js
+const string = 'This is a string'
+```
+
 - An array is a type that can hold multiple variables inside itself. Arrays are defined by using the square brackets `[]`
-- Object are basically advanced arrays, you will learn more about it later in this tutorial. Their defined by curly brackets `{}` 
+
+```js
+const array = [1, 2, 3]
+```
+- Object are basically advanced arrays, you will learn more about it later in this tutorial. Their defined by curly brackets `{}`
+
+```js
+const object = {}
+```
+
 - Functions are also their own type.
+
+```js
+const function = (a, b) => {return a+b}
+```
+
 - A boolean is a type that can only be `true` or `false`
+
+```js
+const boolean = true
+```
+
 - When something is not (yet) defined, its type is `undefined`
 
 ```js
-let number = 1                  // Number type
-let string = 'This is a string' // String type
-let array = [1, 2, 3]           // Array type
-let object = {}                 // Object type
-let function = () => {}         // Function type
-let boolean = true              // Boolean type
-let nothing                     // Undefined type
+const nothing
```

Comment:
```js
// Both variables will be undefined
const nothing
const notDefined = undefined
```

Reply:
Added

Reply:
Seems to be lacking the comment. Someone may miss the point covered by it

Reply:
I didn't want to use a comment because the comment was explained in a later chapter, but I guess I could just move the comment section higher up.